app/views/directives/osc-webhook-triggers.html

```diff
@@ -0,0 +1,82 @@
+<ng-form name="secretsForm" class="add-webhook">
+  <div ng-repeat="trigger in webhookTriggers">
+    <div class="add-webhook-row">
+      <ui-select 
+        ng-model="trigger.data.type"
+        ng-disabled="isDeprecated(trigger)"
+        on-select="triggerTypeChange(trigger)"
+        on-remove="test()"
+        search-enabled="false"
+        title="Select a webhooke type"
+        class="select-webhook-type"
+        flex>
+        <ui-select-match placeholder="Webhook type">
+          {{ $select.selected.label }}
+        </ui-select-match>
+        <ui-select-choices repeat="option.label as option in webhookTypesOptions">
+          {{ option.label }}
+        </ui-select-choices>
+      </ui-select>
+
+      <ui-select
+        ng-if="!isDeprecated(trigger)"
+        ng-model="trigger.data[trigger.data.type.toLowerCase()].secretReference.name"
+        ng-disabled="!trigger.data.type"
+        ng-required="trigger.data.type"
+        on-select="triggerSecretChange(trigger)"
+        title="Select a webhooke secret reference"
+        class="select-secret-ref"
+        flex>
+        <ui-select-match placeholder="Webhook secret reference">
+          {{ $select.selected.metadata.name }}
+        </ui-select-match>
+        <ui-select-choices repeat="webhookSecret.metadata.name as webhookSecret in (webhookSecrets | filter : $select.search)">
+          <div ng-bind-html="webhookSecret.metadata.name | highlight : $select.search"></div>
+        </ui-select-choices>
+      </ui-select>
+
+      <div class="select-secret-ref deprecated-secret input-group" ng-if="isDeprecated(trigger)">
+        <input
+          ng-model="trigger.data[trigger.data.type.toLowerCase()].secret"
+          class="form-control"
+          type="password"
+          autocorrect="off"
+          autocapitalize="none"
+          spellcheck="false"
+          select-on-focus
+          disabled>
+        <span class="input-group-btn">
+          <a
+              role="button"
+              href=""
+              class="btn btn-default"
+              title="Toggle Password Visibility"
+              aria-label="Toggle Password Visibility"
+              ng-click="togglePassword($index)">
+            <i class="glyphicon glyphicon-eye-open"></i>
```

Comment:
aria-hidden="true"

We probably shouldn't use `glyphicon`

Reply:
I think glyphicons are fine since they ship with Bootstrap/PatternFly.